tests/parser/features/iteration/test_range_in.py

```diff
@@ -71,4 +71,33 @@ def testin() -> bool:
     return False
     """
     with pytest.raises(TypeMismatchException):
-        c = get_contract(code)
+        get_contract(code)
+
+
+def test_ownership():
+    code = """
+
+owners: address[10]
```

Comment:
only 2 addresses needed to show functionality below

Reply:
:+1: